src/mkdocs_autorefs/references.py

```diff
@@ -164,9 +165,11 @@ def inner(match: Match):  # noqa: WPS212,WPS430
                 return f"[{identifier}][]"
             return f"[{title}][{identifier}]"
 
+        classes = "autorefs "
+        classes += "external" if urlparse(url).scheme else "internal"
```

Comment:
```suggestion
        classes = "autorefs"
        classes += " external" if urlparse(url).scheme else " internal"
```
because.. imagine there are more possible additions or if they are optional